pcweb/pages/docs/hosting/self_hosting.py

```diff
@@ -7,22 +7,43 @@
     docpage,
     doctext,
     subheader,
+    docalert,
 )
 
-code_example1 = """rx.text('hello world', color='blue')"""
-code_example2 = """
-rx.hstack(
-    rx.circular_progress(
-        rx.circular_progress_label("50", color="green"),
-        value=50,
-    ),
-    rx.circular_progress(
-        rx.circular_progress_label("∞", color="rgb(107,99,246)"),
-        is_indeterminate=True,
-    ),
+hosting_config = """config = rx.Config(
+    app_name="your_app_name",
+    api_url="http://192.168.1.1:8000",
+    bun_path="$HOME/.bun/bin/bun",
+    db_url="sqlite:///reflex.db",
+)
+"""
+
+docker_config = """
+config = rx.Config(
+    app_name="app",
+    api_url="0.0.0.0:8000",
+    bun_path="/app/.bun/bin/bun",
+    db_url="sqlite:///reflex.db",
 )
 """
 
+project_structure = """hello
+├── .web
+├── assets
+├── hello
+│   ├── __init__.py
+│   └── hello.py
+├── rxconfig.py
+├── Dockerfile
+└── requirements.txt"""
+
+docker_build_example = """$ docker build -t reflex-project:latest ."""
+docker_run_example = """$ docker run -d -p 3000:3000 -p 8000:8000 --name reflex reflex:latest"""
```

Comment:
```suggestion
docker_run_example = """$ docker run -d -p 3000:3000 -p 8000:8000 --name reflex reflex-project:latest"""
```

the image to run should be the same tag as the image built in the previous step